spec/models/card_spec.rb

```diff
@@ -11,23 +11,25 @@
     end
   end
 
-  describe '#add_review_date' do
+  context '#add_review_date' do
     it 'equals time now + 3days' do
       expect(card.review_date).to eq(Date.today + 3.days)
     end
   end
 
-  describe '#prepare_word' do
+  context '#prepare_word' do
     it 'work with cyrillic' do
       expect(card).not_to be_valid
     end
   end
 
-  describe '#check_answer' do
+  context '#check_answer' do
     let(:answer) { 'Привет' }
 
     it 'user\'s answer equals to original text' do
       expect(answer).to eq(card.original_text)
     end
   end
+
+  context ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.